hypervisor/src/common.rs

```diff
@@ -16,3 +16,8 @@ pub use {
     kvm_bindings::kvm_vcpu_events as VcpuEvents, kvm_ioctls::DeviceFd, kvm_ioctls::IoEventAddress,
     kvm_ioctls::VcpuExit,
 };
+
```

Comment:
It would be good to amend the commit message as to briefly describe why this is needed.